ui/tests/acceptance/open-api-path-help-test.js

```diff
@@ -80,7 +80,7 @@ function secretEngineHelper(test, secretEngine) {
       assert.deepEqual(
         Object.keys(result).sort(),
         Object.keys(expected).sort(),
-        `getProps returns expected attributes for ${modelName}`
+        `getProps returns expected attributes for ${modelName} (help url: "${helpUrl}")`
```

Comment:
The help url returns the open API endpoint returned which is useful for finding API docs (our ember data model names don't always map 1:1 to API resources)